test/e2e/framework/test_context.go

```diff
@@ -241,4 +241,15 @@ func ViperizeFlags() {
 
 	// TODO Consider wether or not we want to use overwriteFlagsWithViperConfig().
 	viper.Unmarshal(&TestContext)
+
+	AfterReadingAllFlags(&TestContext)
+}
+
+// AfterReadingAllFlags makes changes to the context after all flags
+// have been read.
+func AfterReadingAllFlags(t *TestContextType) {
```

Comment:
Sorry...My bad...Actually should be [here](https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/e2e_node_suite_test.go#L72), after `Parse`.